templates/http-rust/content/Cargo.toml

```diff
@@ -17,7 +17,6 @@ bytes = "1"
 http = "0.2"
 # The Spin SDK.
 spin-sdk = { git = "https://github.com/fermyon/spin", tag = "v1.3.0" }
-# Crate that generates Rust Wasm bindings from a WebAssembly interface.
```

Comment:
This deletion is only ok if the spin-sdk is aligned with the changes made in this PR. I'm unsure if this should be left for the bump of the SDK version or if it's ok to do this deletion now.